### ISSUE TITLE
Add GitHub Codespaces configuration

### ISSUE THREAD

**Issue by gdomingu** [rubyforgood/pet-rescue#161]
Add devcontainer configuration to the repository to make it easy to open and dev on the repo inside of a Codespace. 

https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/adding-a-dev-container-configuration/introduction-to-dev-containers

**Comment by gdomingu** [comment]
Created this issue to go with PR.

**Comment by kasugaijin** [comment]
Not currently needed for MVP.

**Comment by gdomingu** [comment]
Just as a follow up, we had a volunteer session at our company where a lot of people who wanted to volunteer for this repo were asking for this. They didn't have have all the deps installed and having codespaces would have allowed them to get up and running within the allotted time given to us to volunteer. It might be a nice to have in the future as it would lower the barrier to entry for volunteers.

**Comment by kasugaijin** [comment]
@gdomingu yes I see what you are saying. If you have an event like that coming up, and/or we hear another request for this we can prioritize it.
